docs/excel2xml-module.md

```diff
@@ -258,23 +258,11 @@ else:
     excel2xml.make_boolean_prop(":hasBoolean", False)
 ```
 
-#### Supported text values 
```

Comment:
I think that text values " " and "-" are only a theoretical concern. I don't need to bother the end user with such subtleties. (The way how excel2xml handles this remains unchanged. I just kicked it out of the docs).